### PR TITLE
PORTAL-3078: Shared Catalogs end-user documentation

### PR DESCRIPTION
**Pull request by manolo317**

### CODE REVIEW

File: content/themes-for-customer.json [hunk 1]
```diff
@@ -23,6 +23,11 @@
             "img": "retailer-shop-corrected.svg",
             "description": "I want to set up my onboarder instance."
         },
+        "manage-shared-catalogs-instance":{
+            "title": "Set up your Shared Catalogs  instance",
+            "img": "retailer-shop-corrected.svg",
```
Comment (manolo317) [inline content/themes-for-customer.json:28]: Need a new image for the menu

File: content/themes-for-system-integrator.json [hunk 2]
```diff
@@ -23,6 +23,11 @@
             "img": "retailer-shop-corrected.svg",
             "description": "I want to set up an Onboarder instance."
         },
+        "manage-shared-catalogs-instance":{
+            "title": "Set up your Shared Catalogs  instance",
```
Comment (Nuscly) [inline content/themes-for-system-integrator.json:27]:
```suggestion
            "title": "Set up a Shared Catalogs  instance",
```

File: content/themes-for-system-integrator.json [hunk 3]
```diff
@@ -23,6 +23,11 @@
...
+            "img": "retailer-shop-corrected.svg",
```
Comment (Nuscly) [inline content/themes-for-system-integrator.json:28]: idem for the svg.

File: content/md/shared-catalogs/1-set-up-shared-catalogs.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+1. Select a `Linked PIM instance` in the list.
+2. Choose an `Instance name`, it will not be a domain name.
+3. Input your `PIM API connection credentials`, test the connection to validate.
+4. Once the form is filled in and connection successfully tested, click on `Request an instance`.
+
+![Shared Catalogs setup form](../img/shared_catalogs_setup.gif)
+
+::: warning
+If the linked PIM is a Flexibility instance it should be compatible only with 4.0 and above version.
+:::
```
Comment (Nuscly) [inline content/md/shared-catalogs/1-set-up-shared-catalogs.md:35]:
```suggestion
::: warning
If the linked PIM is a Flexibility instance, the Shared catalogs are compatible only with PIM version 4.0 and above.
:::
```

**Review by docteurklein — APPROVED** [review]

File: content/md/shared-catalogs/1-set-up-shared-catalogs.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+![Shared Catalogs list](../img/shared_catalogs_list.png)
+
+1. First, from the menu, go to `Project console / Projects` and select your project.
+2. Go to the Shared Catalogs instance tab, and open the instance with the status `Waiting for setup`, by clicking on the instance line.
+3. Then fill in the form with the missing prerequisites in order to create the Shared Catalogs instance.
+
+## Link your Shared Catalogs to an existing PIM Cloud instance
+
+1. Select a `Linked PIM instance` in the list.
```
Comment (alan-akeneo-2020) [inline content/md/shared-catalogs/1-set-up-shared-catalogs.md:26]: You may add that it will pre-fill the installation continent field?

**Pull request merged**